.github/workflows/publish_docker_images.yml

```diff
@@ -0,0 +1,45 @@
+name: ci
```

Comment:
should we use more descriptive names for out workflows?

Reply:
What do you suggest? Same as the filename `publish_docker_images`?

Reply:
how does "publish image on docker hub" sounds?